### PR TITLE
feat(docs): add spanish translation for glossary page

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do ?

- Resolves adds new spanish definitions to glossary page

### Steps to test 
1. Run `yarn build`
2. Run `yarn start`
3. Go to `localhost:3000`
4. Go to Glossary page

### CODE REVIEW

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 1]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
 
 ## **A**
 
+### Cuenta
```
Comment (xavier506) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:13]: Cuenta deberia estar bajo la `C`

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 2]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
 
-### API (Interfaz de Programación de Aplicaciones) 
+Un objeto que tiene una dirección, un saldo, un nonce y posiblemente también algo de almacenamiento y código. Una cuenta puede ser una cuenta de contrato o una cuenta de propiedad externa (EOA).
 
-Conjunto de funciones y procedimientos que permiten la creación de aplicaciones que acceden a las características o datos de un sistema operativo, aplicación u otro servicio especifico.
+---
+
+### Dirección
```
Comment (xavier506) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:19]: place under `D`

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 3]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
 
-
-* * *
+---
 
 ## **B**
 
 ### Backend
 
 Backend es la capa de acceso a datos de un software, que no es directamente accesible por los usuarios,contiene la lógica de la aplicación que maneja dichos datos. El Backend accede al servidor, que es una aplicación especializada que entiende la forma como el navegador solicita información.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:23]: I would change "navegador" for a more generic term "software cliente"

**Review by kuronosec — CHANGES_REQUESTED** [review]

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 4]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
 
-* * *
+---
 
 ### Billeteras
 
 Las billeteras son clientes que almacenan llaves privadas asociadas con los permisos de una o más cuentas. Idealmente, una billetera tiene un estado bloqueado (encriptado) y desbloqueado (sin encriptar) que está protegido por una contraseña de alta entropía.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:29]: Encriptado ==> Cifrado

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 5]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
 
-* * *
+---
 
 ### Blacklist (Lista negra)
 
-Una lista negra es una medida que un nodo de la cadena de bloques puede poner en práctica para impedir que bloquee cualquier transacción que haya sido autorizada por una lista definida de cuentas bloqueados o cualquier transacción que interactúe con una lista definida de contratos. Cada nodo puede definir su propia lista negra, que no se puede ver públicamente. 
+Una lista negra es una medida que un nodo de la cadena de bloques puede poner en práctica para impedir que bloquee cualquier transacción que haya sido autorizada por una lista definida de cuentas bloqueados o cualquier transacción que interactúe con una lista definida de contratos. Cada nodo puede definir su propia lista negra, que no se puede ver públicamente.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:35]: Not clear to me, too confusing

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 6]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
 
-* * *
+---
 
-### Blockhain 
+### Blockhain
 
 Es una lista creciente de registros, llamados bloques, que están vinculados mediante criptografía. Cada bloque contiene un hash criptográfico del bloque anterior, una marca de tiempo y datos de transacción (generalmente representados como un árbol Merkle).
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:41]: de transacciones

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 7]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
-Se define como el número de bloques en la cadena entre un bloque dado y el primer bloque en la cadena de bloques. Por ejemplo una altura de bloque de 500 se refiere al bloque número 500 de esa cadena. 
+Se define como el número de bloques en la cadena entre un bloque dado y el primer bloque en la cadena de bloques. Por ejemplo una altura de bloque de 500 se refiere al bloque número 500 de esa cadena.
 
-* * *
+---
 
-### Bloque 
+### Bloque
 
 Los bloques son archivos donde los datos pertenecientes a la cadena de bloques se registran de forma permanente. Almacenan algunas o todas las transacciones más recientes que aún no han ingresado a ningún bloque anterior. Un bloque es un almacén permanente de registros que una vez escritos no pueden ser alterados o eliminados.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:59]: archivos --> conjuntos de registros

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 8]
```diff
@@ -10,149 +10,213 @@ Lista de términos necesarios para comprender un poco mas los contenidos present
...
+
+### Ether
+
+La criptomoneda nativa que utiliza el ecosistema Ethereum y que se utiliza para pagar las tarifas de gas cuando se ejecutan las transacciones.
+
+---
+
+### Estado
+
+Un estado instantáneo de todos los saldos y datos en un momento determinado en la cadena de bloques, que generalmente se refiere a la condición en un bloque específico.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:137]: Don't use "Estado" as the beginning of the definition of the word "Estado"

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 9]
```diff
@@ -162,131 +226,221 @@ Los `issues` son una excelente manera de realizar un seguimiento de las tareas,
...
+
+Verificar que un nuevo bloque tiene transacciones y firmas válidas, se basa en la cadena histórica más pesada y se adhiere a todos los demás requisitos de consenso. Los bloques válidos se agregan al final de la cadena y se distribuyen a la red, donde se descartan los bloques no válidos.
+
+---
+
 ## **W**
 
 ### Wallet
 
 Las `Wallets` son clientes que almacenan llaves privadas asociadas con los permisos de una o más cuentas. Idealmente, una billetera tiene un estado bloqueado (encriptado) y desbloqueado (sin encriptar) que está protegido por una contraseña de alta entropía.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:435]: Cifrado

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 10]
```diff
@@ -162,131 +226,221 @@ Los `issues` son una excelente manera de realizar un seguimiento de las tareas,
...
+
+---
 
 ## **U**
 
 ## **V**
 
+### Validador
+
+Un nodo es responsable de almacenar datos, ejecutar transacciones y agregar nuevos bloques a la cadena de bloques en un sistema de prueba de participación. Para que el software de validación se active, se deben apostar 32 ETH.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:421]: apostar --> depositar en la cadena

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 11]
```diff
@@ -162,131 +226,221 @@ Los `issues` son una excelente manera de realizar un seguimiento de las tareas,
...
 ### TPS (Transacciones por Segundo)
 
 Una métrica mediante la cual se mide el rendimiento de una cadena de bloques.
 
-* * * 
+---
+
+### Tarifa base
+
+The reserve price for each block is known as the "base fee." It is the minimum gas fee a user must pay for a transaction to be included in the subsequent block.
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:405]: This is in English

File: docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md [hunk 12]
```diff
@@ -162,131 +226,221 @@ Los `issues` son una excelente manera de realizar un seguimiento de las tareas,
...
+Un lenguaje de programación procedimental que tiene una sintaxis comparable a JavaScript, C++ y Java. El lenguaje más utilizado y popular para los contratos inteligentes de Ethereum.
+
+### Sincronizando
+
+El proceso de descargar la última versión completa de una cadena de bloques en un nodo.
+
+### Susurro
+
+Un servicio de mensajería descentralizado (P2P). Se usa junto con Web3 y Swarm para construir DApps.
+
```
Comment (kuronosec) [inline docs/i18n/es/docusaurus-plugin-content-docs/current/research/glossary.md:374]: This a name, don't traslate it

**Review by kuronosec — APPROVED** [review]
LGTM

**Pull request merged**